SKR PRO DIN Mount

0lympu5 opened the pull request:
<!--
Thank you for your interest in contributing to the VoronUsers repository, it is
highly appreciated!

**Please make sure the submission conforms to the rules outlined below. PRs which fail
to conform to the rules below are likely to be rejected.**
-->

  * [ x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x ] Submitted STL files are oriented and scaled properly for printing.
  * [ x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

<!--
Describe the submission further using the template provided below. The more 
details the better!
-->

#### Which mods/configurations/profiles are added by this PR?
Mount for SKR PRO boards
#### How was it tested? 
Tested and Fitted to my machine
#### Any background context you want to provide?
NO
#### Screenshots (if appropriate)
One within the readme.md file
#### Further notes
No

KiloQubit reviewed (changes requested):
Thanks for your mod!

- Please create a subdirectory for your mod (e.g. `printer_mods/MrBean66/SKR_PRO_Mount/` and move your files in there
- Please edit `printer_mods/README.md` to add a link to your mod in alphanumeric order
- Please remove spaces from your filenames

0lympu5 commented on printer_mods/MrBean66/README.md:
```diff
@@ -0,0 +1,14 @@
+- 4x M3 x 6 BHCS
+- 4x M2 self tapers 
+
+![Image of mod](https://github.com/0lympu5/VoronUsers/blob/master/printer_mods/MrBean66/SKR%20PRO%20Example.png)
```
Hi there, thanks for the guidance and ill change appropriately. Will try and add a separate file for this mod however it refused to upload when I created the proper folder tree.

I'm confused as to what you mean by alphanumeric link to the mod

KiloQubit commented:
Let me see if I can fix the directories for you.

KiloQubit commented:
Ok, I fixed your directory structure, removed spaces from filenames, and removed the empty file.  Please update your README image with a relative link like I mentioned above, and update `printer_mods/README.md` to add a link to your mod.  See the table in that file - find the place in the table where your username would be in alphabetical order (alphanumeric is alphabetical + numbers), and insert a row there - basically copy an existing row and modify it with your name, mod name, like and compatibility checkboxes.  Looks like your mod should go in the table after `Mosher`.

0lympu5 commented:
Thank you, any issues and ill have another look

On Fri, 9 Apr 2021, 2:36 am KiloQubit, ***@***.***> wrote:

> Ok, I fixed your directory structure, removed spaces from filenames, and
> removed the empty file. Please update your README image with a relative
> link like I mentioned above, and update printer_mods/README.md to add a
> link to your mod. See the table in that file - find the place in the table
> where your username would be in alphabetical order (alphanumeric is
> alphabetical + numbers), and insert a row there - basically copy an
> existing row and modify it with your name, mod name, like and compatibility
> checkboxes. Looks like your mod should go in the table after Mosher.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/VoronDesign/VoronUsers/pull/276#issuecomment-816340326>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AK4RADXX24G6WKX5M24YJHLTHZKY3ANCNFSM42RLAKFQ>
> .
>

0lympu5 commented:
Okay, I believe all the requested changes have been made.  Hope everything should be set

KiloQubit reviewed (approved):
Thanks for the updates!
